feconf.py

```diff
@@ -1008,3 +1009,11 @@ def get_empty_ratings():
 
     }
 }
+
+# A dict representing available landing pages, having subject as a key and list
+# of topics as the value.
+AVAILABLE_LANDING_PAGES = {
+    'maths': ['fractions', 'ratios']
```

Comment:
How will you ensure that this matches the JS?

Reply:
I didn't find any way except adding a top-level comment.